File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        unordered_set<int> numset(nums.begin(), nums.end());
        int res = 1;
        if (nums.empty()){
            return 0;
        }
        for (int n : numset){
            if (!numset.contains(n-1)){
                int length = 1;
                while (numset.contains(n+length)){
                    length++;
                    res = max(res, length);
                }
            }
        }
        return res;


    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-2.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        mc = {}
        res = []
        for i, n in enumerate(nums):
            mc[n] = mc.get(n, 0) + 1
        
        while k != 0:
            max_key = max(mc, key=mc.get)
            res.append(max_key)
            mc.pop(max_key)
            k -= 1
        
        return res